app/components/channel_icon/dm_avatar/dm_avatar.tsx

```diff
@@ -24,11 +25,24 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
     statusInfo: {
         backgroundColor: theme.centerChannelBg,
     },
+    icon: {
+        color: changeOpacity(theme.sidebarText, 0.4),
```

Comment:
What about find channels? The dolor may differ

Reply:
Also - there are a few places where the `ProfilePicture` component is called as an icon; worth returning the archived icon from there? Though it would mean any place that the archived users avatar / profile photo is shown it'll return the archived icon.

Reply:
Nope, just for channel items.

As for the color, not sure what you are thinking, is basically passing the `isInfo` prop from the ChannelIcon to this component and apply the correct color.